reference/5.1/Microsoft.PowerShell.Core/About/about_Character_Encoding.md

```diff
@@ -0,0 +1,230 @@
+following values:
+
+- `ascii`: Uses the encoding for the ASCII (7-bit) character set.
+- `bigendianunicode`: Encodes in UTF-16 format using the big-endian byte order.
+- `oem`: Uses the default encoding for MS-DOS and console programs.
+- `unicode`: Encodes in UTF-16 format using the little-endian byte order.
+- `utf7`: Encodes in UTF-7 format.
+- `utf8`: Encodes in UTF-8 format.
+- `utf8BOM`: Encodes in UTF-8 format with Byte Order Mark (BOM)
+- `utf8NoBOM`: Encodes in UTF-8 format without Byte Order Mark (BOM)
+- `utf32`: Encodes in UTF-32 format.
+
+PowerShell Core defaults to `utf8NoBOM` for all output.
+
+Beginning with PowerShell 6.2, the **Encoding** parameter also allows numeric
+IDs of registered code pages (like `-Encoding 1251`) or string names of
+registered code pages (like `-Encoding "windows-1251"`). For more information,
+see the .NET documentation for
+[Encoding.CodePage](/dotnet/api/system.text.encoding.codepage?view=netcore-2.2&preserve-view=true).
+
+## Changing the default encoding
+
+PowerShell has two default variables that can be used to change the default
+encoding behavior.
+
+- `$PSDefaultParameterValues`
+- `$OutputEncoding`
+
+For more information, see
+[about_Preference_Variables](about_Preference_Variables.md).
+
+Beginning in PowerShell 5.1, the redirection operators (`>` and `>>`) are
+effectively aliases of `Out-File`. You can set the default encoding them using
+the `$PSDefaultParameterValues` preference variable.
+
+You can set the default encoding for redirection and `Out-File` using the
+following statement.
+
+```powershell
+$PSDefaultParameterValues['Out-File:Encoding'] = 'utf8'
+```
+
+Use the following statement to change the default encoding for all cmdlets that
+have the **Encoding** parameter.
+
+```powershell
+$PSDefaultParameterValues['*:Encoding'] = 'utf8'
+```
+
+> [!IMPORTANT]
+> Putting this command in your PowerShell profile makes the preference a
+> session-global setting that affects all commands and scripts that do not
+> explicitly specify an encoding.
+>
+> Similarly, you should include such commands in your scripts or modules that
+> you want to behave the same way. Using these commands ensure that cmdlets
+> behave the same way even when run by another user, on a different computer,
+> or in a different version of PowerShell.
+
+The automatic variable `$OutputEncoding` affects the encoding PowerShell uses
+to communicate with external programs. It has no effect on the encoding that
+the output redirection operators and PowerShell cmdlets use to save to files.
+
+## See also
+
+- [Code Pages - Win32 apps](/windows/win32/intl/code-pages)
+- [Introduction to character encoding in .NET](/dotnet/standard/base-types/character-encoding-introduction)
+- [The Unicode Standard](https://www.unicode.org/standard/standard.html)
+- [Encoding.CodePage](/dotnet/api/system.text.encoding.codepage?view=netcore-2.2&preserve-view=true)
```

Comment:
Same question here on hardlocking to 2.2.

Reply:
Fixed